wgpu/src/api/compute_pipeline.rs

```diff
@@ -48,6 +55,25 @@ pub struct ComputePipelineDescriptor<'a> {
     /// Debug label of the pipeline. This will show up in graphics debuggers for easy identification.
     pub label: Label<'a>,
     /// The layout of bind groups for this pipeline.
+    ///
+    /// If this is set, then `wgpu` will validate that the layout matches what the shader module(s)
+    /// expect, otherwise [`Device::create_compute_pipeline`] will panic.
```

Comment:
```suggestion
    /// expect, otherwise [`Device::create_compute_pipeline`] will cause a validation error.
```

Should only panic with the default error handler.
(see also https://github.com/gfx-rs/wgpu/issues/3767)